src/components/layout/Navbar.js

```diff
@@ -231,19 +232,44 @@ ServiceListEntry.propTypes = {
 const ServiceList = () => {
   const [services, isLoading, isError] = useFetchServiceExtensions()
   const [open, setOpen] = React.useState(true)
+  const navigate = useNavigate()
 
-  const handleClick = () => {
+  const handleToggleClick = (event) => {
+    event.stopPropagation()
     setOpen(!open)
   }
 
+  const handleToggleKeyDown = (event) => {
+    if (event.key === 'Enter' || event.key === ' ') {
+      event.stopPropagation()
+      setOpen(!open)
+    }
+  }
+
+  const handleNavigationClick = () => {
+    navigate(`${SERVICES_PATH}`)
```

Comment:
nit: you can just directly call `navigate(SERVICES_PATH)` (there is no need to explicitly parse as string since `SERVICES_PATH` is already a string)